fe/fe-core/src/main/java/org/apache/doris/datasource/jdbc/JdbcExternalCatalog.java

```diff
@@ -379,10 +378,11 @@ private void testJdbcConnection() throws DdlException {
 
     private void testFeToJdbcConnection() throws DdlException {
         try {
+            initLocalObjectsImpl();
```

Comment:
Why move this method to here?

Reply:
Initializing the client in the method that tests FE's external connection is more convenient for process control, rather than outside this method